extensions/wikia/WikiaInYourLang/WikiaInYourLangController.class.php

```diff
@@ -114,7 +114,7 @@ public function getWikiDomain( $sCurrentUrl ) {
 
 		if ( isset( $aParsed['host'] ) ) {
 			$sHost = $aParsed['host'];
-			$regExp = "/(sandbox.{3}\.|preview\.|verify\.)?(([a-z]{2,3}|[a-z]{2}\-[a-z]{2})\.)?([^\.]+\.)([^\.]+\.)(.*)/i";
+			$regExp = "/(sandbox-.+?|preview|verify)\.?(([a-z]{2,3}|[a-z]{2}\-[a-z]{2})\.)?([^\.]+\.)([^\.]+\.)(.*)/i";
```

Comment:
What kind of fix was applied here?

If we fixed something, maybe we could extending unit tests to cover the cases that were previously broken?